privacy/blocklists/quantum-adlist.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Quantum Ad-list",
+  "website": "https://gitlab.com/The_Quantum_Alpha/the-quantum-ad-list",
+  "description": "It is a list containing over 1300000 domains used by ads, trackers, miners, malwares, and much more!",
+  "source": {
```

Comment:
Why don't you add the individual list here instead of the global one ?
If you add the 3 individual it will let more control to NextDns user to choose what they want to block with these lists.